service/src/integrationTest/kotlin/fi/espoo/evaka/daycare/controllers/UnitAclControllerIntegrationTest.kt

```diff
@@ -77,6 +90,19 @@ class UnitAclControllerIntegrationTest : FullApplicationTest() {
         assertTrue(getAclRows().isEmpty())
     }
 
+    @Test
+    fun `supervisor message account`() {
+        assertEquals(employeeMessageAccountState(), MessageAccountState.NO_ACCOUNT)
+        insertSupervisor(testDaycare.id)
+        assertEquals(employeeMessageAccountState(), MessageAccountState.ACTIVE_ACCOUNT)
+        insertSupervisor(testDaycare2.id)
+        assertEquals(employeeMessageAccountState(), MessageAccountState.ACTIVE_ACCOUNT)
+        deleteSupervisor(testDaycare.id)
+        assertEquals(employeeMessageAccountState(), MessageAccountState.ACTIVE_ACCOUNT)
+        deleteSupervisor(testDaycare2.id)
+        assertEquals(employeeMessageAccountState(), MessageAccountState.INACTIVE_ACCOUNT)
```

Comment:
These should be flipped `(expected, actual)`

Reply:
> Do we implement `where active = TRUE` filtering in a separate PR?

Yes, added a ticket to miro